components/style/servo/media_queries.rs

```diff
@@ -267,5 +278,23 @@ lazy_static! {
             keyword_evaluator!(eval_scan, Scan),
             FeatureFlags::empty(),
         ),
+        feature!(
+            atom!("resolution"),
+            AllowsRanges::Yes,
+            Evaluator::Resolution(eval_resolution),
+            FeatureFlags::empty(),
+        ),
+        feature!(
+            atom!("device-pixel-ratio"),
+            AllowsRanges::Yes,
+            Evaluator::Float(eval_device_pixel_ratio),
+            FeatureFlags::WEBKIT_PREFIX,
+        ),
+        feature!(
+            atom!("-moz-device-pixel-ratio"),
```

Comment:
The compat spec has ` -webkit-device-pixel-ratio` and not `-moz-device-pixel-ratio`, so maybe it would be better to add the former instead? Not sure

Reply:
I copied that from mozilla: https://searchfox.org/mozilla-central/source/servo/components/style/gecko/media_features.rs#751 and webkit version is also added if I understand `WEBKIT_PREFIX` correctly (that one is also copied from mozilla).